portal-frontend/src/app/features/notifications/edit-submission/transferees/transferees.component.ts

```diff
@@ -22,12 +23,15 @@ export class TransfereesComponent extends StepComponent implements OnInit, OnDes
   displayedColumns: string[] = ['type', 'fullName', 'organizationName', 'phone', 'email', 'actions'];
 
   private submissionUuid = '';
+  isMobile = false;
+  VISIBLE_COUNT = 5;
```

Comment:
As a constant, I would move this outside of the component and define it with `const`. I'd also change the name to something like `VISIBLE_COUNT_INCREMENT`.

Reply:
Moved and renamed the constants to constants.ts file in shared module. (Same as alcs-frontend)

Reply:
Nice. This is even better.